### ISSUE TITLE
error: unknown command 'setup-ios-permissions' when running react-native setup-ios-permissions

### ISSUE THREAD

**Issue by felipejoribeiro** [zoontek/react-native-permissions#801]
### Bug summary

When I run the `react-native setup-ios-permissions` command, I get the following output:
```
$ react-native setup-ios-permissions

warn Package @sentry/react-native contains invalid configuration: "dependency.platforms.ios.sharedLibraries" is not allowed,"dependency.hooks" is not allowed. Please verify it's properly linked using "react-native config" command and contact the package maintainers about this.
error: unknown command 'setup-ios-permissions'
```

### Library version

3.6.1

### Environment info

```shell
$ react-native info                                        

warn Package @sentry/react-native contains invalid configuration: "dependency.platforms.ios.sharedLibraries" is not allowed,"dependency.hooks" is not allowed. Please verify it's properly linked using "react-native config" command and contact the package maintainers about this.
info Fetching system and libraries information...
System:
  OS: macOS 13.3
  CPU: (8) arm64 Apple M1 Pro
  Memory: 69.33 MB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 16.15.0
    path: ~/.nvm/versions/node/v16.15.0/bin/node
  Yarn:
    version: 1.22.19
    path: ~/.nvm/versions/node/v16.15.0/bin/yarn
  npm:
    version: 8.5.5
    path: ~/.nvm/versions/node/v16.15.0/bin/npm
  Watchman:
    version: 2023.08.28.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.12.1
    path: /Users/fejori/.rbenv/shims/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.1
      - iOS 16.1
      - macOS 13.0
      - tvOS 16.1
      - watchOS 9.1
  Android SDK:
    API Levels:
      - "23"
      - "29"
      - "30"
      - "31"
      - "32"
      - "33"
    Build Tools:
      - 29.0.2
      - 29.0.3
      - 30.0.2
      - 30.0.3
      - 31.0.0
      - 32.0.0
      - 32.1.0
      - 33.0.0
    System Images:
      - android-27 | Google APIs ARM 64 v8a
      - android-29 | Google Play ARM 64 v8a
      - android-31 | Google Play ARM 64 v8a
      - android-32 | Google APIs ARM 64 v8a
    Android NDK: Not Found
IDEs:
  Android Studio: 2021.1 AI-211.7628.21.2111.8309675
  Xcode:
    version: 14.1/14B47b
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.18
    path: /usr/bin/javac
  Ruby:
    version: 2.7.4
    path: /Users/fejori/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli":
    installed: 7.0.4
    wanted: 7.0.4
  react:
    installed: 17.0.2
    wanted: 17.0.2
  react-native:
    installed: 0.68.1
    wanted: 0.68.1
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: false
  newArchEnabled: false
iOS:
  hermesEnabled: false
  newArchEnabled: false
```

### Steps to reproduce

1. run `react-native setup-ios-permissions`

### Reproducible sample code

```js
#!/bin/bash

react-native setup-ios-permissions
```

**Comment by zoontek** [comment]
@felipejoribeiro Try npx / yarn react-native setup-ios-permissions. Here it just cannot locate the react native CLI.

**Comment by felipejoribeiro** [comment]
I tried the commands without success:
```
$ yarn react-native setup-ios-permissions
yarn run v1.22.19
$ /Users/fejori/Work/snackin/mobile-app/node_modules/.bin/react-native setup-ios-permissions
error Unrecognized command "setup-ios-permissions".
info Run "react-native --help" to see a list of all available commands.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
and 

```
$ npx react-native setup-ios-permissions
error Unrecognized command "setup-ios-permissions".
info Run "react-native --help" to see a list of all available commands.
```

**Comment by zoontek** [comment]
@felipejoribeiro Ah, got it, you are not using the latest version. Please update, the command was not available yet in 3.6.1.

https://github.com/zoontek/react-native-permissions/releases/tag/3.7.0

The master branch README is the latest version documentation. Please refers to tags to get your version README: https://github.com/zoontek/react-native-permissions/tags

**Comment by felipejoribeiro** [comment]
Thank you! Cheers
